fern/pages/changelogs/docs/2024-12-30.mdx

```diff
@@ -0,0 +1,7 @@
+## Support for /sitemap.xml
+
+Added a `/sitemap.xml` file to improve search engine indexing and site visibility.
+
+### Why it matters:
+- Ensures all key pages are easily discoverable by search engines.
+- Speeds up indexing for new and updated pages, improving SEO and user access.
```

Comment:
@kgowru fern's had a sitemap.xml for a while (anything else we can substitute this with) ?